core/src/main/kotlin/okuri/core/documentation/blocks/design.md

```
@@ -0,0 +1,652 @@
+
+Blocks can be laid out and structured in a number of different ways to support maximum flexibility.
+
+## Blocks
+
+### Content Blocks
+
+Content blocks hold renderable data directly within their payload. This data is validated against the block type's
+schema, and can be directly interacted and edited with.
+All content blocks have an associated ComponentNode within it's BlockType to define how it is rendered in the frontend.
+
+**High Level Design**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)